godotproject/Objects/Sawblade/SawBladeArea.tscn

```diff
@@ -20,8 +20,8 @@ script = ExtResource( 4 )
 
 [node name="SawBlade" type="KinematicBody2D" parent="."]
 z_index = -1
-collision_layer = 64
-collision_mask = 0
+collision_layer = 68
+collision_mask = 4
```

Comment:
What collision layer is this adding?

Reply:
we talked about this in standup but for record: adds enemy collision layer so rockets will explode on collision